vnext/Microsoft.ReactNative/Views/ScrollViewManager.cpp

```diff
@@ -500,6 +525,27 @@ XamlView ScrollViewManager::CreateViewCore(int64_t /*tag*/, const winrt::Microso
   return scrollViewer;
 }
 
+void ScrollViewManager::SetLayoutProps(
+    ShadowNodeBase &nodeToUpdate,
+    const XamlView &viewToUpdate,
+    float left,
+    float top,
+    float width,
+    float height) {
+  // ScrollViewer selects an anchor during the Arrange phase of layout.
+  // If you do not call InvalidateArrange whenever a new child is added
+  // to the ScrollViewer content, the anchor behavior does not work.
```

Comment:
How would this work outside of RN scenarios? Would default ScrollView child layout invalidate the ScrollView?

Reply:
I believe it's something like that, yes. I imagine that if you looked at the implementation for https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.itemsupdatingscrollmode?view=winrt-20348 you'd find that the ItemsStackPanel calls InvalidateArrange on the ScrollViewer parent as part of it's layout lifecycle.